.github/workflows/release.yml

```
@@ -54,5 +55,5 @@ jobs:
           version: pnpm run version
           publish: pnpm run publish
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.VERSION_PACKAGES_TOKEN }}
```

I was thinking, could we name this:
```suggestion
          GITHUB_TOKEN: ${{ secrets.CHANGESETS_GITHUB_TOKEN }}
```

---

Interestingly, Spectacle does have a variable named `CHANGESETS_GITHUB_TOKEN` too, but I don't see it being used anywhere.
Spectacle doesn't have "Required PR Checks" enabled, so they can merge without running any checks.

---

agree with the name change - will change this up!